contents/docs/session-replay/_snippets/react-native-privacy.mdx

```diff
@@ -1,5 +1,37 @@
+# Masking
```

Comment:
I'd copy-pasta the code snippets from android, ios and flutter as well.
this is gonna be the 3rd place where everything is duplicated though.

would it make sense to link it back to https://posthog.com/docs/session-replay/installation?tab=React%20Native#step-three-configure-replay-settings instead so we avoid the dupe?

or the other way around?

Reply:
I want to do a pass over the privacy docs properly soon. The problem with linking back without duping is then you have to grok the whole thing to pull out what you care about. But privacy is such a specific task within setup/management